query-engine/driver-adapters/executor/src/wasm.ts

```diff
@@ -3,28 +3,23 @@ import * as wasmMysql from '../../../query-engine-wasm/pkg/mysql/query_engine_bg
 import * as wasmSqlite from '../../../query-engine-wasm/pkg/sqlite/query_engine_bg.js'
 import fs from 'node:fs/promises'
 import path from 'node:path'
-import { fileURLToPath } from 'node:url'
-
-const dirname = path.dirname(fileURLToPath(import.meta.url))
+import { __dirname } from './utils'
 
 const wasm = {
     postgres: wasmPostgres,
     mysql: wasmMysql,
     sqlite: wasmSqlite
 }
 
-type EngineName = keyof typeof wasm;
+type EngineName = keyof typeof wasm
 
 const initializedModules = new Set<EngineName>()
 
-
-
 export async function getEngineForProvider(provider: EngineName) {
     const engine = wasm[provider]
     if (!initializedModules.has(provider)) {
         const subDir = provider === 'postgres' ? 'postgresql' : provider
-        const bytes = await fs.readFile(path.resolve(dirname, '..', '..', '..', 'query-engine-wasm', 'pkg', subDir, 'query_engine_bg.wasm'))
-        console.error(bytes)
+        const bytes = await fs.readFile(path.resolve(__dirname, '..', '..', '..', 'query-engine-wasm', 'pkg', subDir, 'query_engine_bg.wasm'))
```

Comment:
Unrelated, but this `console.error(bytes)` leftover has been bugging me for the past month